spk/sabnzbd/src/service-setup.sh

```diff
@@ -8,7 +8,7 @@ LANGUAGE="env LANG=en_US.UTF-8"
 
 GROUP="sc-download"
 
-SERVICE_COMMAND="${LANGUAGE} ${PYTHON} ${SABNZBD} -f ${CFG_FILE} --pidfile ${PID_FILE} -d"
+SERVICE_COMMAND="${LANGUAGE} ${PYTHON} -OO ${SABNZBD} -f ${CFG_FILE} --pidfile ${PID_FILE} -d"
```

Comment:
@Safihre LGTM but what does `-OO` do?

Reply:
It runs in "optimized" mode, so potentially a tiny bit faster.
https://docs.python.org/3/using/cmdline.html#cmdoption-o